### PR TITLE
Add express-validation v4 API for all methods

### PR DESCRIPTION
**Pull request by hamishwillee**

### CODE REVIEW

File: controllers/bookController.js [hunk 1]
```diff
@@ -86,84 +88,83 @@ exports.book_create_get = function(req, res, next) {
...
+    // Sanitize fields
+    sanitizeBody('title').trim().escape(),
+    sanitizeBody('author').trim().escape(),
+    sanitizeBody('summary').trim().escape(),
+    sanitizeBody('isbn').trim().escape(),
+    
+    // Process request after validation and sanitization
+    (req, res, next) => {
+        
+        // Sanitize genre array for each value individually (validator only works for strings)
```
Comment (hamishwillee) [inline controllers/bookController.js]: @VaibhavAjayGupta I used the legacy API here. I would appreciate your advice on the best way to do this on the new API!

**Comment by hamishwillee** [comment]
@VaibhavAjayGupta This follows on from your #25. I'm going to merge it later today, but please do feel free to review and comment. There is also a question inline for you :-)

File: controllers/bookController.js (hunk 1 above)
Comment (VaibhavAjayGupta) [reply controllers/bookController.js]: My suggestion is #31

**Comment by VaibhavAjayGupta** [comment]
Thank you!

**Pull request merged**